test/support/fixtures/webinars_fixtures.ex

```diff
@@ -20,6 +21,9 @@ defmodule MembraneLive.WebinarsFixtures do
     {:ok, webinar} =
       attrs
       |> Enum.into(@default_webinar_attrs)
+      |> Map.update!("moderators", fn moderators ->
+        if user.email in moderators, do: moderators, else: [user.email | moderators]
+      end)
```

Comment:
Isn't it the same line as in `MembraneLive.Webinars.create_webinar`? Also shouldn't we guarantee that creator's email is always inside the moderator's list?

Reply:
This line was excessive - we are adding the creator to moderators in `create_webinar` already.